tests/components/cover/test_zwave.py

```diff
@@ -8,58 +8,61 @@
     MockNode, MockValue, MockEntityValues, value_changed)
 
 
-def test_get_device_detects_none(mock_openzwave):
+def test_get_device_detects_none(hass, mock_openzwave):
     """Test device returns none."""
     node = MockNode()
     value = MockValue(data=0, node=node)
     values = MockEntityValues(primary=value, node=node)
 
-    device = zwave.get_device(node=node, values=values, node_config={})
+    device = zwave.get_device(hass, node=node, values=values, node_config={})
```

Comment:
Let's use keyworded arguments here to match the invocation we're using in `zwave/__init__.py` https://github.com/turbokongen/home-assistant/blob/59dcb12a77da73f815f4e7b513d5123606b85310/homeassistant/components/zwave/__init__.py#L726